Bottom: 3326ff15b329545c866e3809bebf072a96d3057d
Top:    37a843b045d17d5316961aa1c771df27c7cf3ccf
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-13 15:45:36 -0500

provision production server


---
diff --git a/provision.sh b/provision.sh
index 3762e89d9..d58ff0fac 100755
--- a/provision.sh
+++ b/provision.sh
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 #PYTHONUNBUFFERED=1 \
-     
+
 ANSIBLE_ENABLE_TASK_DEBUGGER=True \
 ANSIBLE_FORCE_COLOR=true \
 ANSIBLE_HOST_KEY_CHECKING=false \
@@ -9,6 +9,6 @@ ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o Contr
 exec ansible-playbook \
         --connection=ssh \
         --timeout=30 \
-        --inventory-file=ansible-playbook/staging \
+        --inventory-file=ansible-playbook/production \
         -vv \
         ansible-playbook/site.yml "$@"
